### PR TITLE
Replaced data directory content, added support for labels patch, quotas

### PR DESCRIPTION
**Pull request by JuozasA**
Replaced data directory content so it is aligned to managed-cluster-config project structure.
added capability for operator to check and patch labels without aditional reformating.
Cluster Resource Quotas can be set in custom resource and then they will be applied to the cluster

### CODE REVIEW

File: pkg/sync/data/sre-authorization/10-osd-ldap-ca-configmap.ConfigMap.yaml [hunk 1]
```diff
@@ -0,0 +1,34 @@
+apiVersion: v1
+data:
+  ca.crt: |-
+    -----BEGIN CERTIFICATE-----
```
Comment (mjudeikis) [inline pkg/sync/data/sre-authorization/10-osd-ldap-ca-configmap.ConfigMap.yaml:4]: WOW. :crying_cat_face:

**Comment by mjudeikis** [comment]
This somehow has corrupted `go.sum` file. Remove it from the PR, and run `go mod download` and it should generate new one with right hash'es

**Comment by mjudeikis** [comment]
https://github.com/openshift/release/pull/7027

**Comment by mjudeikis** [comment]
CI is broken little bit now. Give it some time :)

**Comment by mjudeikis** [comment]
I think there is still an issue with generate content if to believe test output

**Comment by JuozasA** [comment]
i think linter does not like those hive patches... anyway, I'll check test locally

/hold

**Comment by JuozasA** [comment]
@rafael-azevedo do you have permissions to merge this PR?

**Pull request closed without merging**